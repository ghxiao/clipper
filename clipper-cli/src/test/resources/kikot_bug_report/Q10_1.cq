Q(?3, ?8) <-  R(?3, ?4), S(?4, ?5), R(?5, ?6), S(?6, ?7), R(?7, ?8)
