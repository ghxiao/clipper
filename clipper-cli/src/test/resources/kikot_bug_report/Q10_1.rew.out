ri(X,Y) :- ga(X,Y).
r(X,Y) :- gb(X,Y).
s(Y,X) :- si(X,Y).
s(X,Y) :- ga(X,Y).
r(Y,X) :- ri(X,Y).
si(Y,X) :- s(X,Y).
si(X,Y) :- gb(X,Y).
ri(Y,X) :- r(X,Y).
